yt_dlp/extractor/bbc.py

```diff
@@ -317,16 +317,26 @@ def _raise_extractor_error(self, media_selection_error):
 
     def _download_media_selector(self, programme_id):
         last_exception = None
+        formats, subtitles = [], {}
         for media_set in self._MEDIA_SETS:
             try:
-                return self._download_media_selector_url(
+                fmts, sttls = self._download_media_selector_url(
                     self._MEDIA_SELECTOR_URL_TEMPL % (media_set, programme_id), programme_id)
+                formats.extend(fmts)
+                if sttls:
+                    subtitles = self._merge_subtitles(subtitles, sttls)
```

Comment:
```suggestion
                self._merge_subtitles(subs, target=subtitles)
```

Reply:
```
self._merge_subtitles(subs, target=subtitles)
```
`_merge_subtitles`  does not accept a non-`None` argument.